### ISSUE TITLE
Accessibility feedback circa May 2026

### ISSUE THREAD

**Issue by antobinary** [bigbluebutton/greenlight#6285]
Thanks @michelleDeko for summarizing and forwarding the following report outlining accessibility areas to be improved in GL

## Greenlight (Landing page)

- Logo has no meaningful alternative text (globally labeled with `alt="CompanyLogo"` — not editable)
- Heading hierarchy (h4 after h1)
- Improve cards markup (headings should be h3, cards should be `article` elements)

## Login form

- H1 missing in the markup
- Missing `autocomplete` attributes for form auto-fill

## Landing page after login

- Missing H1
- Markup architecture is very non-semantic on the cards (use `article` elements)
- No consistently visible focus indicator on the Rooms tab, and missing accessible name on the URL copy button
- "Start" buttons should be more distinguishable for screen reader users via additional info (visually hidden)
- Card is clickable and then links to the room settings (not accessible by keyboard alone, since it is only operable via JS and mouse)

Additional points
- Head title always the same [regardless of room?] (BigBlueButton)
- Missing accessible routing?
- Non-text contrast is insufficient (form fields, checkboxes, switches, buttons, etc. — 1.3:1, 3:1)
- Icon markup produces unnecessary noise for screen reader users (insufficient markup)
